script/build-icons

```diff
@@ -1,11 +1,28 @@
 #!/bin/sh
 
-dir=$(cd $(dirname $0) && cd "../src/common/icons" && pwd)
-src="${dir}/icon.svg"
+# script/build-icons: Build optimized png icons in all required sizes and
+#                     colors from the svg icons.
+#
+#                     Running this script requires `inkscape` and `pngcrush`
+#                     executables in your PATH.
+
+set -o errexit
+set -o nounset
+
+dir="$(realpath "$(dirname "$0")/../src/common/icons")"
+dark_src="${dir}/icon.svg"
+light_src="${dir}/icon-light.svg"
+
+export_png_icon() {
+  src=$1 dest=$2
+  inkscape -z -o "$dest" -D -w "$s" "$src" # export
+  convert "$dest" -background none -gravity center -extent "${s}x${s}" "$dest" # pad
+  pngcrush -brute -ow "$dest" # reduce
+}
 
 for s in 16 32 48 64 128; do
-  dest="${dir}/icon-${s}.png"
-  inkscape -z -e $dest -D -w $s $src # export
-  convert $dest -background none -gravity center -extent "${s}x${s}" $dest # pad
-  pngcrush -brute -ow $dest # reduce
+  dark_dest="${dir}/icon-${s}.png"
+  light_dest="${dir}/icon-light-${s}.png"
+  export_png_icon "$dark_src" "$dark_dest"
+  export_png_icon "$light_src" "$light_dest"
```

Comment:
The size doesn't seem to be passed as a third argument yet? 👀

Reply:
Oops. Maybe I was too early and you were still working no it? Sorry 😬

Reply:
Was in a super long meeting, sorry. Should have put a `WIP` label... 🙈